Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift

```diff
@@ -330,12 +330,27 @@ extension SubscriptionsViewController: UITableViewDelegate {
         return view
     }
 
+    func tableView(_ tableView: UITableView, willSelectRowAt indexPath: IndexPath) -> IndexPath? {
+        if let selected = tableView.indexPathForSelectedRow {
+            tableView.deselectRow(at: selected, animated: true)
+        }
+        return indexPath
+    }
+
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let subscription = self.groupSubscriptions?[indexPath.section][indexPath.row]
         let controller = ChatViewController.sharedInstance()
         controller?.closeSidebarAfterSubscriptionUpdate = true
         controller?.subscription = subscription
     }
+
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        guard let sell = cell as? SubscriptionCell else { return }
```

Comment:
`cell` is `UITableViewCell` but not `SubscriptionCell` so it did not have property `subscription`.